.eslintrc

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "canonical"
+  "extends": "canonical",
+  "rules": {
+    "promise/prefer-await-to-callbacks": "off",
```

Comment:
No need to disable the rule across the project. Use `// eslint-disable-next-line promise/prefer-await-to-callbacks`

Reply:
Ok - there's a few of them, but I don't think it ends up being too cluttered.